bitnami/etcd/values.yaml

```diff
@@ -113,6 +113,16 @@ allowNoneAuthentication: true
 ##
 # maxProcs:
 
+# Auto Compaction Parameters
+#
+# auto compaction mode , by default periodic
+# autoCompactionMode: periodic
+#
+# auto compaction retention is interpred on the basis of auto compaction mode
+# by default 0 , means disabled
+# autoCompactionRetention: 0
+#
```

Comment:
A suggestion about the comments format to follow the same approach used for other parameters:
```suggestion
## Auto compaction mode, by default periodic. Valid values: ‘periodic’, ‘revision’. 
## - 'periodic' for duration based retention, defaulting to hours if no time unit is provided (e.g. ‘5m’). 
## - 'revision' for revision number based retention.
##
# autoCompactionMode: periodic

## Auto compaction retention for mvcc key value store in hour, by default 0, means disabled
##
# autoCompactionRetention: 0
```